README.md

```diff
@@ -0,0 +1,34 @@
+# Daala next-generation video codec
+
+Daala is the codename for a new video compression technology. The effort is a
+collaboration between the Mozilla Foundation, the Xiph.Org Foundation and any
+other contributors that wish to help. The goal of the project is to provide a
+video format that's free to implement, use and distribute, and a reference
+implementation with technical performance superior to H.265. 
+For more information, see the [Daala wiki][wiki].
+
+To get a sense of how well Daala performs in comparison to other codecs, check
+out [Are We Compressed Yet?][awcy].
+
+[awcy]: https://arewecompressedyet.com/
+[wiki]: https://xiph.org/daala/
+![Tiger barfing rainbows](https://people.xiph.org/~greg/a.png)
+
+## Hacking on Daala
```

Comment:
I'd probably put these links even before the others. If someone has the code, they want to hack.

I'd also add the other links from <https://mailarchive.ietf.org/arch/msg/hackathon/PnbblctpV-YrP5ASgXVgu-lumsk>, specifically the issue tracker, code review, and continuous integration links.